src/modules/conversation-handler/conversationGpt.ts

```diff
@@ -44,7 +56,23 @@ export async function conversationGpt(
         if (response) {
           chat.push({ content: response.completion, role: "system" });
           usage += response.usage;
-          ctx.reply(response.completion!);
+          price = response.price;
+          totalPrice += price
+          console.log(usage, totalPrice);
+          await ctx.reply(response.completion!);
+          conversation.session.openAi.chatGpt.chatConversation = [...chat];
+          console.log(response.completion, usage, price);
+          const isPay = await conversation.external(() => {
+            return payments.pay(ctx as OnMessageContext, price);
+          });
+          console.log("here", isPay);
```

Comment:
Remove console.log or create more clear log output and use logger

Reply:
Deleted. I was using the console.log as code flags.